src/app/home/home.component.ts

```diff
@@ -374,15 +378,21 @@ export class HomeComponent implements AfterViewInit {
     }
 
 
-    getTextOfTextMarking(i: number) {
-        const editor = document.getElementById(this.EDITOR_KEY)!;
+    getTextOfTextMarking(i: number): string {
+        const editor: HTMLElement | null = document.getElementById(this.EDITOR_KEY)!;
```

Comment:
I guess the `!` at the end of this line should be dropped

Reply:
👍